api/test/fakes/register.js

```diff
@@ -0,0 +1,12 @@
+let count = 0;
+
+module.exports = (overrides) => {
+  count += 1;
```

Comment:
I dont understand the count also isnt the definition outside of scope?

Reply:
Its in the module scope which is available to the function. It's a really easy way to make sure that we don't have any conflicts.